pandas/tests/frame/test_join.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import pandas as pd
+
+from pandas.tests.frame.common import TestData
+
+import pandas.util.testing as tm
+from pandas.util.testing import (assertRaisesRegexp,
```

Comment:
I actually prefer NOT to import the testing routines and instead use ``tm.assert_frame_equal``, but as long as its consistent one way of the other is ok.